examples/testing-dom__page-source/cypress/e2e/spec.cy.js

```diff
@@ -1,6 +1,11 @@
 /// <reference types="cypress" />
 
 describe('Page source', () => {
+  // ignore errors from the site itself
+  Cypress.on('uncaught:exception', () => {
+    return false
+  })
```

Comment:
Should we throw this in `setupNodeEvents` so they're always ignored? Or is it preferred in this case to only ignore based on the spec?

Reply:
Ah just read your description! Maybe would be good to include that context so other peeps know that we want to remove these ignores once the errors are resolved

```
//  Ignore errors from the site itself until they're fixed
```

Reply:
I'll keep them in the individual specs, but I updated the handling to only ignore these specific exceptions and fleshed out the comment a bit.